plop-templates/component.js.hbs

```diff
@@ -0,0 +1,4 @@
+angular.module('portainer.{{module}}').component('{{camelCase name}}', {
+  templateUrl: './{{camelCase name}}.html',
+  controller: '{{properCase name}}Controller'
+});
```

Comment:
From above comment
> + in the future we should be able to leverage only the ES6 class on `component.js` file, so iirc we won't need to register the class as Controller in AngularJS.

Here we should be able to transform to
```
import {{properCase name}}Controller from './{{properCase name}}.controller.js'; // (or any file pattern we will have)

angular.module('portainer.{{module}}').component('{{camelCase name}}', {
  templateUrl: './{{camelCase name}}.html',
  controller: {{properCase name}}Controller
});
```
Registration should be automatic and previous controller registration should be obsolete.

Reply:
yep, it's feasible to do right now. I like it